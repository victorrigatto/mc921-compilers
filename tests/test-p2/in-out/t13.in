void main(int i) {
  assert(i > 0);
}
