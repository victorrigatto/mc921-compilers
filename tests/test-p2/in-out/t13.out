Program:
    FuncDef:
        Type: void @ 1:1
        Decl: ID(name=main)
            FuncDecl:
                ParamList:
                    Decl: ID(name=i)
                        VarDecl:
                            Type: int @ 1:11
                VarDecl:
                    Type: void @ 1:1
        Compound: @ 1:18
            Assert: @ 2:3
                BinaryOp: > @ 2:10
                    ID: i @ 2:10
                    Constant: int, 0 @ 2:14
